src/hotspot/share/runtime/perfData.hpp

```diff
@@ -417,6 +417,7 @@ class PerfLongVariant : public PerfLong {
     inline void inc() { (*(jlong*)_valuep)++; }
     inline void inc(jlong val) { (*(jlong*)_valuep) += val; }
     inline void dec(jlong val) { inc(-val); }
+    inline void reset() { (*(jlong*)_valuep) = 0; }
```

Comment:
This new function doesn't seem to be used.

Reply:
Removed.